scopes/docs/docs/docs.main.runtime.ts

```diff
@@ -132,6 +135,16 @@ export class DocsMain {
     return this.patterns;
   }
 
+  getComponentDevPatterns(component: Component) {
+    const env = this.envs.calculateEnv(component).env;
+    const componentEnvDocsDevPatterns: string[] = env.getDocsDevPatterns ? env.getDocsDevPatterns(component) : [];
```

Comment:
you should add `getDocsDevPatterns` to the [environments interfaces](https://github.com/teambit/bit/blob/master/scopes/envs/envs/environment.ts)